node_modules/oae-activity/lib/internal/transformer.js

```diff
@@ -46,7 +46,15 @@ var transformActivities = module.exports.transformActivities = function(ctx, act
             _getActivityEntitiesByObjectType(activityId, activity.target, activityEntitiesByObjectType);
         });
     } catch (err) {
-        log().error({'err': err, 'activities': activities, 'user': ctx.user()}, 'Failed to get activity entities');
+        var logActivities = [];
+        activities.forEach(function(activity) {
+            logActivities.push({
+                'activityActor': activity.actor.id,
+                'activityObject': activity.object.id,
+                'activityTarget': activity.target.id
+            });
+        });
```

Comment:
We typically use underscore for our collections operations. [`_.map`](http://underscorejs.org/#map) can be used in this case to transform the entities. Also, I don't think the keys need to be prefixed with `activity` here (e.g., `activityActor`) as we are nesting this object in an `activities` object in the log.